packages/core/util/functionStrings.ts

```diff
@@ -1,14 +1,17 @@
+import createJexlInstance from './jexl'
+
 export const functionRegexp = /^\s*function\s*\w*\s*\(([^)]*)\)\s*{([\w\W]*)/
 
-const compilationCache: Record<string, Function> = {}
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const compilationCache: Record<string, any> = {}
 
 /**
- * compile a function to a string
+ * compile a jexlExpression to a string
  *
- * @param str - string of code like `function() { ... }`
+ * @param str - string of code like `jexl:...`
  * @param options -
```

Comment:
fork this as `jexlStrings.ts` and leave this file as-is, we might want to use it later in a more limited way